exasol/toolbox/tools/security.py

```diff
@@ -0,0 +1,266 @@
+"""This module contains security related CLI tools and code"""
+import json
+import re
+import subprocess
+import sys
+from dataclasses import (
+    asdict,
+    dataclass,
+)
+from enum import Enum
+from functools import partial
+from inspect import cleandoc
+from typing import (
+    Generator,
+    Iterable,
+    Tuple,
+)
+
+import typer
+
+stdout = print
+stderr = partial(print, file=sys.stderr)
+
+
+# Note: In the long term we may want to adapt the official CVE json schema
```

Comment:
Ticket?

Reply:
the official schema seems excessive complicated

Reply:
> the official schema seems excessive complicated

Agreed, that's one reason why I rather would avoid it (if not required or practical).
@redcatbear  mention though that we should use `std` formats, that's why
I just referenced it. If it makes sense at some point in the future ...
For now I wanted to have an `MVP` which we can iterate on, and use to learn what we need.